src/main/java/com/dev/cinema/security/implementations/AuthenticationServiceImpl.java

```diff
@@ -24,7 +24,7 @@ public AuthenticationServiceImpl(UserService userService,
 
     @Override
     public User login(String email, String password) throws AuthenticationException {
-        Optional<User> userFromDB = userService.findByEmail(email);
+        Optional<User> userFromDB = Optional.ofNullable(userService.findByEmail(email));
```

Comment:
inside `findByEmail()` method in service you do the following:

`userDao.findByEmail(email).get();`

what happens if we call optional.get() on empty optional? will we return null or what?

Reply:
done